api4/user.go

```diff
@@ -42,6 +43,8 @@ func InitUser() {
 	BaseRoutes.User.Handle("/sessions", ApiSessionRequired(getSessions)).Methods("GET")
 	BaseRoutes.User.Handle("/sessions/revoke", ApiSessionRequired(revokeSession)).Methods("POST")
 	BaseRoutes.User.Handle("/audits", ApiSessionRequired(getAudits)).Methods("GET")
+
+	BaseRoutes.Users.Handle("/autocomplete/", ApiSessionRequired(autocompleteUsers)).Methods("GET")
```

Comment:
from mux/route.go: 

/ PathPrefix adds a matcher for the URL path prefix. This matches if the given
// template is a prefix of the full URL path. See Route.Path() for details on
// the tpl argument.
//
// Note that it does not treat slashes specially ("/foobar/" will be matched by
// the prefix "/foo") so you may want to use a trailing slash here.

I think that's why we should use a '/' after autocomplete